packages/usa-file-input/src/index.js

```diff
@@ -109,19 +110,64 @@ const makeSafeForID = (name) => name.replace(/[^a-z0-9]/g, replaceName);
 const createUniqueID = (name) =>
   `${name}-${Math.floor(Date.now().toString() / 1000)}`;
 
+/**
+ * Builds an element that will house the file upload status
+ * @param {HTMLElement} fileInputEl - original file input on page
+ * @param {HTMLElement} fileInputParent - file input wrapper with the DROPZONE_CLASS
+ * @param {HTMLElement} instructions - text to inform users to drag or select files
+ * @param {HTMLElement} dropTarget - target area div that encases the input
+ */
+const buildStatusMessage = (
+  fileInputEl,
+  fileInputParent,
+  instructions,
+  dropTarget
+) => {
+  const statusEl = document.createElement("div");
+  const acceptsMultiple = fileInputEl.hasAttribute("multiple");
+  const instructionsEl = instructions;
+  const items = "files";
+  let item = "file";
+  let chooseText = "";
+  let defaultInstructionsText = "";
+  let defaultStatus = "";
+  let dragText = "";
+
+  // Set up status message and add it to the DOM
+  statusEl.classList.add(SR_ONLY_CLASS);
+  statusEl.setAttribute("aria-live", "polite");
+  fileInputParent.insertBefore(statusEl, dropTarget);
+
+  // If multiple attribute enabled, use the plural
+  if (acceptsMultiple) {
+    item = items;
+  }
+
+  // Add initial file status message
+  defaultStatus = `No ${item} selected.`;
+  statusEl.setAttribute("data-default-status-text", defaultStatus);
+  statusEl.textContent = defaultStatus;
+
+  // Add initial instructions for input usage
+  dragText = `Drag ${item} here or`;
+  chooseText = "choose from folder";
+  defaultInstructionsText = `${dragText} ${chooseText}`;
+  fileInputEl.setAttribute("aria-label", defaultInstructionsText);
+  fileInputEl.setAttribute("data-default-aria-label", defaultInstructionsText);
```

Comment:
Is this just to store the original value? We could avoid modifying markup by saving it in JS at the start of the function.

```suggestion
  const defaultAriaLabel = fileInputEl.getAttribute("aria-label"); // defaultInstructionsText
```

Reply:
I wasn't sure how to pull this across functions, into `handleChange()`. Might be just a brain hiccup. Maybe we can get on a call tomorrow?